packages/ckeditor5-dev-env/lib/release-tools/changelog/writer-options.js

```diff
@@ -101,14 +105,22 @@ function transformCommit( commit ) {
 
 	commit.type = getCommitType( commit.type );
 
-	if ( commit.scope === '*' ) {
-		commit.scope = '';
-	}
-
 	if ( typeof commit.subject === 'string' ) {
 		commit.subject = linkGithubIssues( linkGithubUsers( commit.subject ), issues );
 	}
 
+	if ( typeof commit.body === 'string' ) {
+		commit.body = commit.body.split( '\n' )
+			.map( ( line ) => {
+				if ( !line.length ) {
+					return line;
```

Comment:
Just return an empty string then. It will be easier to read because you won't force people to analyze the code around.